src/components/Sections/GroupedSectionsView.tsx

```diff
@@ -29,11 +28,7 @@ export const GroupedSectionView = ({
 
       <div
         className={cx(
-          'shortcuts-list u-w-100 u-mh-auto u-flex-justify-center',
-          {
-            'shortcuts-list--gutter': isTwakeTheme(),
-            'u-mv-3 u-mv-2-t ': !isTwakeTheme()
-          }
+          'shortcuts-list shortcuts-list--gutter u-w-100 u-mh-auto u-mv-3 u-mv-2-t u-flex-justify-center'
```

Comment:
`u-mv-3 u-mv-2-t` comes from `!isTwakeTheme()`, is it ok? 🤔

Reply:
It is okay because it is not display for Twake users